### PR TITLE
refactor: Remplace Input component by UI-kit Input

### LINKED ISSUE #438: Remove the input component
Remove https://github.com/pancakeswap/pancake-frontend/blob/develop/src/components/Input/Input.tsx and replace it by the Input in the uikit 
https://pancakeswap.github.io/pancake-uikit/?path=/story/components-input--default

### PR DESCRIPTION
**Pull request by rickmort**
This PR remplaces the Input component by the Input UI-Kit component.

Delete the Input.tsx
Update BalanceInput.tsx directory and imports

closes #438

### CODE REVIEW

**Review by rickmort — COMMENTED** [review]
I decided to create a folder for the BalanceInput component.

And so remove the directory entirely from Input

File: src/components/BalanceInput/BalanceInput.tsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 import useI18n from 'hooks/useI18n'
-import Input from './Input'
 
-interface BalanceInputProps extends BoxProps {
+export interface BalanceInputProps extends BoxProps {
   title: string
   max: number
   symbol: string
-  onChange: (evt: React.FormEvent<HTMLInputElement>) => void
+  onChange?: (evt: React.FormEvent<HTMLInputElement>) => void
```
Comment (hachiojidev) [inline src/components/BalanceInput/BalanceInput.tsx:10]: What is the reason why `onChange` is optional?

File: src/components/TicketInput/TicketInput.tsx [hunk 2]
```diff
@@ -1,37 +1,34 @@
...
 import useI18n from '../../hooks/useI18n'
-import Input, { InputProps } from '../Input'
 
 interface TokenInputProps extends InputProps {
   max: number | string
   symbol: string
   availableSymbol: string
+  value: string
   onSelectMax?: () => void
+  onChange?: (evt: React.FormEvent<HTMLInputElement>) => void
```
Comment (hachiojidev) [inline src/components/TicketInput/TicketInput.tsx:12]: Same here.

File: src/components/TokenInput/TokenInput.tsx [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+import { Button, Flex, Input, InputProps } from '@pancakeswap-libs/uikit'
 import useI18n from '../../hooks/useI18n'
-import Input, { InputProps } from '../Input'
 
 interface TokenInputProps extends InputProps {
   max: number | string
   symbol: string
+  value: string
   onSelectMax?: () => void
+  onChange?: (evt: React.FormEvent<HTMLInputElement>) => void
```
Comment (hachiojidev) [inline src/components/TokenInput/TokenInput.tsx:11]: Additionally, why are we typing this? It is already included in `InputProps`...

File: src/components/BalanceInput/BalanceInput.tsx (hunk 1 above)
Comment (rickmort) [reply src/components/BalanceInput/BalanceInput.tsx]: No, sorry
Fixed

File: src/components/TokenInput/TokenInput.tsx (hunk 3 above)
Comment (rickmort) [reply src/components/TokenInput/TokenInput.tsx]: The ui-kit InputProps type only has scale, isSuccess and is isWarning props.

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**